### PR TITLE
Fix: remove cartography from InputBarButtonsView

### PR DESCRIPTION
**Pull request by billypchan**
## What's new in this PR?

 remove cartography from InputBarButtonsView

### CODE REVIEW

**Comment by KaterinaWire** [comment]
@billypchan There are SwiftLint issues

**Comment by billypchan** [comment]
@KaterinaWire fixed lint issues.

**Comment by KaterinaWire** [comment]
> @KaterinaWire fixed lint issues.

There are a couple new issues

File: Wire-iOS/Sources/UserInterface/Conversation/InputBar/InputBarButtonsView.swift [hunk 1]
```diff
@@ -154,72 +159,106 @@ final class InputBarButtonsView: UIView {
...
         let filled = secondRow.count == numberOfButtons
         let referenceButton = firstRow.count > 1 ? firstRow[1] : firstRow[0]
-        constrainRowOfButtons(secondRow, inset: constants.buttonsBarHeight, rowIsFull: filled, referenceButton: referenceButton)
+
+        constraints.append(contentsOf: constrainRowOfButtons(secondRow, inset: constants.buttonsBarHeight, rowIsFull: filled, referenceButton: referenceButton)
+        )
     }
 
-    fileprivate func constrainRowOfButtons(_ buttons: [UIButton],
+    private func constrainRowOfButtons(_ buttons: [UIButton],
```
Comment (billypchan) [inline Wire-iOS/Sources/UserInterface/Conversation/InputBar/InputBarButtonsView.swift:180]:
```suggestion
        private func constrainRowOfButtons(_ buttons: [UIButton],
```

**Comment by billypchan** [comment]
fixed again

**Review by agisilaos — APPROVED** [review]

**Review by KaterinaWire — APPROVED** [review]

**Pull request merged**